solution/2200-2299/2210.Count Hills and Valleys in an Array/Solution.java

```diff
@@ -0,0 +1,18 @@
+class Solution {
+    public int countHillValley(int[] nums) {
+        int count = 0;
+        for(int i = 1,j = 0; i < nums.length-1; i++) {
```

Comment:
推荐的代码格式

---

Recommended code format
```suggestion
        for (int i = 1, j = 0; i < nums.length - 1; i++) {
```